gp-settings.php

```diff
@@ -137,6 +142,35 @@ function gp_activate_plugin() {
 }
 register_activation_hook( GP_PLUGIN_FILE, 'gp_activate_plugin' );
 
+/**
+ * Run the plugin de-activation code.
+ *
+ * @since    0.1
+ */
+function gp_deactivate_plugin() {
+	// flush the rewrite rule option so it will be re-generated next time the plugin is activated.
+	update_option( 'gp_rewrite_rule', $match_regex );
```

Comment:
Copy/Paste error, it's being set to a blank string so that during the next init (aka the next plugin activation) the rewrite rules will be flushed again.